Lib/site.py

```diff
@@ -558,6 +575,11 @@ def main():
     setquit()
     setcopyright()
     sethelper()
+    # Detect REPL by prompt, -i flag or PYTHONINSPECT env var being set.
+    if (hasattr(sys, 'ps1') or sys.flags.interactive or
+        os.environ.get('PYTHONINSPECT')
+    ):
```

Comment:
Does such formatting conform PEP 8?

Reply:
It does now :).

Thanks for re-review. Both points addressed.

Reply:
Actually, PEP 8 suggests
```python
    if (hasattr(sys, 'ps1') or sys.flags.interactive or
        os.environ.get('PYTHONINSPECT')):
```
or
```python
    if (hasattr(sys, 'ps1') or sys.flags.interactive
            or os.environ.get('PYTHONINSPECT')):
```
https://www.python.org/dev/peps/pep-0008/#indentation

Reply:
Ah ok. I was using `pycodestyle`. Altered to match pep 8 document.